test/Tests/src/Network/Http_Spec.enso

```diff
@@ -309,7 +309,7 @@ spec =
                   "headers": {
                     "Content-Length": "16",
                     "Content-Type": "application/json",
-                    "User-Agent": "Java-http-client/17.0.6"
+                    "User-Agent": "Java-http-client/17.0.7"
```

Comment:
I suggest to replace line:
```ruby
res.body.decode_as_json.should_equal expected_response
```
with
```ruby
simpleBody = res.body.replace_all 'User-Agent: *Java-http-client/.*$' 'User-Agent: Java-http-client`
simpleBody.decode_as_json.should_equal expected_response
```

Reply:
Good idea. But I am struggling too much now with this replacement. It does not work out of the box and I don't want to spend too much time on that. Leaving that one be for now.